src/status_im2/contexts/communities/home/view.cljs

```diff
@@ -72,20 +68,75 @@
          :icon :i/info}
         (i18n/label :t/error)])]))
 
+(def home-communities (memoize render-communities-segments))
+
+(defn communities-header
+  [selected-tab padding-top]
+  [:<>
+   [rn/view {:style {:padding-vertical 8}}
+    [quo/discover-card
+     {:on-press            #(rf/dispatch [:navigate-to :discover-communities])
+      :title               (i18n/label :t/discover)
+      :description         (i18n/label :t/whats-trending)
+      :accessibility-label :communities-home-discover-card}]]
+   [community-segments selected-tab padding-top]])
+
+(defn home-page-comunity-lists
+  [{:keys [selected-tab padding-top]}]
+  (fn []
+    [rn/view {:style {:flex 1}}
+     [communities-header selected-tab padding-top]
+     [home-communities selected-tab]]))
+
+(defn home-sticky-header
+  [{:keys [selected-tab scroll-height padding-top]}]
+  (fn []
+    (when (> @scroll-height 110)
+      [blur/view
+       {:blur-amount   32
+        :blur-type     :xlight
+        :overlay-color (if platform/ios? colors/white-opa-70 :transparent)
+        :style         style/blur-tabs-header}
+       [community-segments selected-tab padding-top]])))
+
+(defn communities-screen-content
+  [selected-tab]
+  (let [scroll-height (reagent/atom 0)]
+    (fn []
+      [scroll-page/scroll-page
+       {:name             (i18n/label :t/communities)
+        :on-scroll        #(reset! scroll-height %)
+        :input-range      (if platform/ios? [-47 10] [0 150])
```

Comment:
what is this input and output range??

Reply:
the height of the blurred slider from the hidden state varies in different screens. its necessary to pass that to the scroll-page component as property

Reply:
these are static values based on the designs, or is it more to do with the screen/phone being used?

Reply:
i see i will roll that back. thanks

Reply:
No it was a question, no need to roll it back.

Reply:
I was curious if we need to make this dynamic or not

Reply:
it can still be as you suggested dynamically updated the background of the other views as the user scroll to still achieve the same effect which appeals better

Reply:
@jo-mut @J-Son89 I would like to note that the `scroll-page` component is using a `scrollview` which we shouldn't be doing for long lists for performance reasons. It should be using a `flatlist`.

Reply:
I agree @OmarBasem. Flatlist would be better for perfomance in this case since the community screen might contain  a long list of items. Maybe there is another approach we can take to still achieve a scrollable screen and a sticky header while using flatlist. This pr uses `map-indexed` to render the list of communities and avoid the warning `VirtualizedLists should never be nested inside plain ScrollViews warning`. I prefer to merge this as is and open an issue that will make use of flatlist in this case

Reply:
@jo-mut sure no problem. You can have sticky headers for different sections using a SecionList component.